Update storage-how-to-create-file-share.md (#124799)
Azure/azure-content: articles/storage/files/storage-how-to-create-file-share.md

| Microsoft.Storage | Pay-as-you-go | HDD (standard) | GeoZone (GZRS) | ![Yes](../media/icons/yes-icon.png) | ![No](../media/icons/no-icon.png) |

## Prerequisites

> - This article assumes that you have an Azure subscription. If you do not have an Azure subscription, then create a [free account](https://azure.microsoft.com/free/?WT.mc_id=A261C142F) before you begin.

Reviewer: ```suggestion
- This article assumes that you have an Azure subscription. If you don't have an Azure subscription, then create a [free account](https://azure.microsoft.com/free/?WT.mc_id=A261C142F) before you begin.
```
